pulse/inngest-router/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:20-alpine
```

Comment:
We generally don't include dockerfiles in this repo. Is this needed?

Reply:
The goal of the example is to be a deployable bundle, rather than an artefact for learning -- so I'd prefer to keep it if it doesn't break too many rules?